include/ctpg.hpp

```diff
@@ -992,9 +992,11 @@ struct parse_options
 {
     constexpr parse_options& set_verbose(bool val = true) { verbose = val; return *this; }
     constexpr parse_options& set_skip_whitespace(bool val = true) { skip_whitespace = val; return *this; }
+    constexpr parse_options& set_dont_skip_newline(bool val = true) { dont_skip_newline = val; return *this; }
 
     bool verbose = false;
     bool skip_whitespace = true;
+    bool dont_skip_newline = false;
```

Comment:
After some thought, this name looks ridiculous. Should be 'skip_newline' defaulting to true.
Of course the value should be ignored when skip_whitespace is false.